### ISSUE TITLE
Maintenance/Support Timeline

### ISSUE THREAD

**Issue by GrantAnt** [census-instrumentation/opencensus-python#1228]
Hello,

I found this article saying that opencensus will be sunsetted. (More information [here](https://opentelemetry.io/blog/2023/sunsetting-opencensus/#fn:1)).
The article also points out the following:
"A number of projects within the opencensus-python repository are still being used as recommended production solutions. These projects will continue to be maintained. For details regarding maintenance timeline, next steps for migration, and general support questions, reach out to repository maintainers."

Can you tell me the current maintenance timeline and/or next steps for migration?

**Comment by lzchen** [comment]
@GrantAnt 

> A number of projects within the opencensus-python repository are still being used as recommended production solutions

This refers specifically to the [opencensus azure monitor exporters](https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-azure). For these exporters, we are now on the path to deprecation officially starting from Oct 2023. We will be supporting them for a year until officially deprecating them. In the meantime, we are directing users to migrate to the [Azure monitor OpenTelemetry Distro](https://learn.microsoft.com/en-us/azure/azure-monitor/app/opentelemetry-enable?tabs=python) which is a solution built on top of OpenTelemetry. Please refer to [this doc](https://learn.microsoft.com/en-us/azure/azure-monitor/app/opentelemetry-python-opencensus-migrate) for migration guidance from opencensus to opentelemetry.

**Comment by radohristov** [comment]
Hello,

there is a new version of flask (3.0.0) which fails to install because of dependency to opencensus-ext-flask (flask < 3.0.0). Can you pls inform me for the plans/timeline to provide compatibility.

Best, Radoslav

**Comment by lzchen** [comment]
@radohristov 

There are no current plans to support flask >3.0.0. Feel free to open up an issue/pr if you are interested.

**Comment by radohristov** [comment]
Yes I'm, because we cant upgrade to Flask 3.X. I will open an issue.
